src/main/java/org/folio/rest/impl/TenantRefAPI.java

```diff
@@ -126,6 +120,19 @@ String servicePointUserFilter(String s) {
               .add("users", "service-points-users");
           }
           return tl.perform(attributes, headers, vertxContext, superRecordsLoaded);
-        });
+        }).onSuccess(handler -> {
```

Comment:
Good point, not sure what's benefit of this because there is no further processing supposed (this part of code possibly will be removed via extra ticket in jira). I'll fix this, thank you Adam.